lib/redis_memo/memoize_query.rb

```diff
@@ -72,26 +72,23 @@ def self.create_memo(model_class, **extra_props)
         )
       end
 
-      extra_props.each do |key, values|
+      extra_props.each do |key, value|
         # The data type is ensured by the database, thus we don't need to cast
         # types here for better performance
         column_name = key.to_s
-        values = [values] unless values.is_a?(Enumerable)
```

Comment:
I don't think we will ever pass in a list of values now. At some point, we can change
```ruby
depends_on Model, ...
```
to be a private API (drop support for passing in a Model in depends_on)

Reply:
I'm still reading through this PR but isn't this done here? https://github.com/chanzuckerberg/redis-memo/blob/main/lib/redis_memo/memoize_query/cached_select.rb#L120

Reply:
I guess what I meant was to hide this API from the users, so they can no longer use https://github.com/chanzuckerberg/redis-memo/blob/main/lib/redis_memo/memoize_query/cached_select.rb#L120